app/views/people/_add_contact.haml

```diff
@@ -3,6 +3,6 @@
   .media
     .pull-right
       = aspect_membership_dropdown(contact, inviter, false)
-    = person_image_link(inviter, :size => :thumb_small, :class => 'img')
+    = person_image_link(inviter, :size => :thumb_small, :class => 'media-left')
     .bd
```

Comment:
When you change this to `.media-body` the username will be to the right of the avatar.